### PR TITLE
feat: rule/assertion message

### LINKED ISSUE #1096: Invalid ruleId for assertions problems
**Describe the bug**
The assertion problems have weird ruleId, it should match the id from the config file.

**To Reproduce**

```yaml
assert/info-description:
  subject:
    type: Info
    property: description
  assertions:
    defined: true
    minLength: 30
    pattern: /(\.|\.\n)$/
```

The problem is reported with ruleId `info-description assertion`.

**Expected behavior**
It should be `assert/info-description`

**Additional context**
There is unneeded complexity in the scorecard code because of this.

### PR DESCRIPTION
**Pull request by Skavengerr**
## What/Why/How?
Added assertion/rule definition as suffix in error message.

Closes #1096

## Reference
https://github.com/Redocly/redocly-cli/issues/1096

## Screenshots (optional)
Before: 
image (1)

After:
Screenshot 2023-06-06 at 18 30 33

## Check yourself

- [x] Code is linted
- [ ] Tested with redoc/reference-docs/workflows
- [x] All new/updated code is covered with tests

## Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

**Review by tatomyr — COMMENTED** [review]
Almost there. Please check my comments.

File: packages/core/src/rules/common/assertions/index.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export const Assertions = (opts: Record<string, Assertion>) => {
 
   for (const [index, assertion] of assertions.entries()) {
     const assertId =
-      (assertion.assertionId && `${assertion.assertionId} assertion`) || `assertion #${index + 1}`;
+      (assertion.assertionId && `${assertion.assertionId}`) || `assertion #${index + 1}`;
```
[the remark this answers is not in the document]
Comment (Skavengerr) [reply packages/core/src/rules/common/assertions/index.ts]: @tatomyr works well

File: packages/core/src/rules/common/__tests__/tag-description.test.ts [hunk 2]
```diff
@@ -57,7 +57,7 @@ describe('Oas3 tag-description', () => {
     const results = await lintDocument({
       externalRefResolver: new BaseResolver(),
       document,
-      config: await makeConfig({ 'tag-description': 'error' }),
+      config: await makeConfig({ 'rule/tag-description': 'error' }),
```
[the remark this answers is not in the document]
Comment (Skavengerr) [reply packages/core/src/rules/common/__tests__/tag-description.test.ts]: deleted, not needed

File: packages/core/src/rules/common/assertions/index.ts [hunk 3]
```diff
@@ -39,9 +39,8 @@ export const Assertions = (opts: Record<string, Assertion>) => {
     (opt: unknown) => typeof opt === 'object' && opt !== null
   );
 
-  for (const [index, assertion] of assertions.entries()) {
-    const assertId =
-      (assertion.assertionId && `${assertion.assertionId} assertion`) || `assertion #${index + 1}`;
+  for (const [_, assertion] of assertions.entries()) {
+    const assertId = assertion.assertionId;
```
Comment (tatomyr) [inline packages/core/src/rules/common/assertions/index.ts:43]: Do we need this const at all? Maybe we can simply use `assertion.assertionId` in place of it?

**Comment by IgorKarpiuk** [comment]
Please link this PR to the issue🙂

**Review by IgorKarpiuk — APPROVED** [review]
LGTM

**Review by tatomyr — APPROVED** [review]

**Pull request merged**